src/test/java/hudson/tasks/AntTest.java

```diff
@@ -123,6 +124,12 @@ private void verify() throws Exception {
         assertEquals(1,l.length);
         r.assertEqualBeans(l[0],new AntInstallation("myAnt","/tmp/foo", JenkinsRule.NO_PROPERTIES),"name,home");
 
+        // Verify that PATH+ANT is set.
```

Comment:
Isn't that already established enough in core? Legitimately wondering here - such a test would not be testing anything specific to Ant, it'd be testing core's `PATH+FOO` logic, which is already covered in https://github.com/jenkinsci/jenkins/blob/master/core/src/test/java/hudson/EnvVarsTest.java#L51-L69